extensions/clockify/src/index.tsx

```diff
@@ -218,26 +260,24 @@ function NewEntry({ updateTimeEntries }: { updateTimeEntries: () => void }) {
 async function getTimeEntries({ onError }: { onError?: (state: boolean) => void }): Promise<TimeEntry[]> {
   const workspaceId = await getLocalStorageItem("workspaceId");
   const userId = await getLocalStorageItem("userId");
+  const { groupTimeEntries } = getPreferenceValues<PreferenceValues>();
 
   const { data, error } = await fetcher(
-    `/workspaces/${workspaceId}/user/${userId}/time-entries?hydrated=true&page-size=500`
+    `/workspaces/${workspaceId}/user/${userId}/time-entries?hydrated=true&page-size=1000`
```

Comment:
@sxn Here the page-size parameter I think is something we need to improve. It affects loading performance but it also depends on the amount of time entries you have and use.

Reply:
yeah, I was thinking about that too. We currently don't support pagination/infinite scrolling though, so we would need to fetch a fixed amount of entries (maybe we can make that a `preference` too?), and then perform subsequent API calls based on whatever the user searches.
WDYT?

Reply:
Yes, I think that could be a preference.

Reply:
Could it be a “how many days” instead? Looks like they have a `start` parameter. If using an input preference field, then the user can pick the number of days he wants – maybe it's just a silly idea, feel free to shoot me down for it 😄

Reply:
that's probably a better idea than just fetching a number of entries 🤔